.env.example

```diff
@@ -0,0 +1,10 @@
+MASTER_PASSWORD=master_password_changeme
+USER_PASSWORD=admin_password_changeme
+APICAST_REGISTRY_URL=https://apicast-staging.proda.3sca.net/policies
+PROMETHEUS_EXPORTER_PORT=9398
+ADMIN_ACCESS_TOKEN=secret
+APICAST_ACCESS_TOKEN=apicastsecret_changeme
```

Comment:
Well the same here and following envs, use simple `token` string ... try to make it easy to setup.